dashboard_templates/InfiniBandNetworkCountersTemplate.json

```diff
@@ -0,0 +1,179 @@
+{
```

Comment:
Yes, this is how they were generated. 
I have not tried importing them using another insights resource; it may be worth trying this today with Yakov.